apps/web/vibes/2px/components/tabs/index.tsx

```diff
@@ -0,0 +1,118 @@
+'use client'
+
+import { CSSProperties, ReactNode, useEffect, useRef, useState } from 'react'
+
+import * as RadixTabs from '@radix-ui/react-tabs'
+import clsx from 'clsx'
+
+type Tab = {
+  title?: string
+  children?: ReactNode
+}
+
+export type Props = {
+  className?: string
+  tabs?: Tab[]
+}
+
+function usePollAnimationFrame(callback: (timestamp: number) => unknown) {
+  useEffect(() => {
+    let requestId: number
+
+    function poll(timestamp: number) {
+      requestId = requestAnimationFrame(poll)
+
+      callback(timestamp)
+    }
+
+    requestId = requestAnimationFrame(poll)
+
+    return () => {
+      cancelAnimationFrame(requestId)
+    }
+  })
+}
+
+const Tabs = ({ className, tabs }: Props) => {
```

Comment:
This is a minor stylistic nit, but can we use function declarations for components? This will also make it consistent with examples:

```ts
export default function Tabs({ className,  tabs }: Props) {
```